tests/yamls/agents/simple_local_agent.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     app: test-example
 spec:
 
-  model: llama3.1
+  model: ollama/llama3.1
```

Comment:
should one of the tests remain without the ollama (so that it tests what happens when no prefix)?